Sources/PaymentButtons/PayPalButton.swift

```diff
@@ -113,69 +113,64 @@ public final class PayPalButton: PaymentButton {
-    
-        private let button: PayPalButton
-        
-        /// Initialize a PayPalButton
-        /// - Parameters:
-        ///   - insets: Edge insets of the button, defining the spacing of the button's edges relative to its content.
-        ///   - color: Color of the button. Default to gold if not provided.
-        ///   - edges: Edges of the button. Default to softEdges if not provided.
-        ///   - size: Size of the button. Default to standard if not provided.
-        ///   - label: Label displayed next to the button's logo. Default to no label.
-        public init(
-            insets: NSDirectionalEdgeInsets? = nil,
-            color: PayPalButton.Color = .gold,
-            edges: PaymentButtonEdges = .softEdges,
-            size: PaymentButtonSize = .standard,
-            label: PayPalButton.Label? = nil,
-            _ action: @escaping () -> Void = { }
-        ) {
-            button = PayPalButton(
-                fundingSource: .payPal,
-                color: color.color,
-                edges: edges,
-                size: size,
-                insets: insets,
-                label: label?.label
-            )
-            self.action = action
-        }
-       
-        // MARK: - UIViewRepresentable methods
-        // TODO: Make unit test for UIVRepresentable methods: https://engineering.paypalcorp.com/jira/browse/DTNOR-623
+/// PayPalButton for SwiftUI
+public struct PayPalButtonView: UIViewRepresentable {
 
-        public func makeCoordinator() -> Coordinator {
-            Coordinator(action: action)
-        }
+    private var action: () -> Void = { }
+
+    private var button: PayPalButton
+
+    /// Initialize a PayPalButton
+    /// - Parameters:
+    ///   - insets: Edge insets of the button, defining the spacing of the button's edges relative to its content.
+    ///   - color: Color of the button. Default to gold if not provided.
+    ///   - edges: Edges of the button. Default to softEdges if not provided.
+    ///   - size: Size of the button. Default to standard if not provided.
+    ///   - label: Label displayed next to the button's logo. Default to no label.
+    public init(
+        insets: NSDirectionalEdgeInsets? = nil,
+        color: PayPalButton.Color = .gold,
+        edges: PaymentButtonEdges = .rounded,
+        size: PaymentButtonSize = .standard,
+        label: PayPalButton.Label? = nil,
+        _ action: @escaping () -> Void = { }
+    ) {
+        button = PayPalButton(
+            fundingSource: .payPal,
+            color: color.color,
+            edges: edges,
+            size: size,
+            insets: insets,
+            label: label?.label
+        )
+        self.action = action
+    }
 
-        public func makeUIView(context: Context) -> UIView {
-            let view = UIView()
+    // MARK: - UIViewRepresentable methods
+    // TODO: Make unit test for UIVRepresentable methods: https://engineering.paypalcorp.com/jira/browse/DTNOR-623
```

Comment:
We don't want links on public GitHub to internal resources:
```suggestion
    // TODO: Make unit test for UIVRepresentable methods: ticket DTNOR-623
```

Reply:
This looks like a bad merge... I didn't type that and the link leads to a 404 error. Will remove